core/oslc4j-jena-provider/pom.xml

```diff
@@ -1,158 +1,153 @@
-<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
-    xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
-    <modelVersion>4.0.0</modelVersion>
-    <parent>
-        <groupId>org.eclipse.lyo.oslc4j.core</groupId>
-        <artifactId>oslc4j-core-build</artifactId>
-        <version>6.0.0-SNAPSHOT</version>
-        <relativePath>../oslc4j-core-build/pom.xml</relativePath>
-    </parent>
-    <artifactId>oslc4j-jena-provider</artifactId>
-    <name>Lyo :: Core :: Jena Provider</name>
-    <description>JAX-RS provider for reading and writing RDF content using the Eclipse Lyo OSLC4J SDK.</description>
+<project xmlns="http://maven.apache.org/POM/4.0.0"
+  xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+  xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+  <modelVersion>4.0.0</modelVersion>
+  <parent>
+    <groupId>org.eclipse.lyo.oslc4j.core</groupId>
+    <artifactId>oslc4j-core-build</artifactId>
+    <version>6.0.0-SNAPSHOT</version>
+    <relativePath>../oslc4j-core-build/pom.xml</relativePath>
+  </parent>
+  <artifactId>oslc4j-jena-provider</artifactId>
+  <name>Lyo :: Core :: Jena Provider</name>
+  <description>JAX-RS provider for reading and writing RDF content using the
+    Eclipse Lyo OSLC4J SDK.</description>
 
-    <properties>
-    </properties>
+  <properties>
+  </properties>
 
-    <dependencies>
-        <dependency>
-            <groupId>org.eclipse.lyo.oslc4j.core</groupId>
-            <artifactId>oslc4j-core</artifactId>
-        </dependency>
+  <dependencies>
+    <dependency>
+      <groupId>org.eclipse.lyo.oslc4j.core</groupId>
+      <artifactId>oslc4j-core</artifactId>
+    </dependency>
 
-        <dependency>
-            <groupId>javax.servlet</groupId>
-            <artifactId>javax.servlet-api</artifactId>
-            <scope>provided</scope>
-        </dependency>
-        <dependency>
-            <groupId>jakarta.ws.rs</groupId>
-            <artifactId>jakarta.ws.rs-api</artifactId>
-            <scope>provided</scope>
-        </dependency>
+    <dependency>
+      <groupId>jakarta.servlet</groupId>
+      <artifactId>jakarta.servlet-api</artifactId>
```

Comment:
I went through all jakarta dependencies, and removed the <provided> scope from all - Except for oauth-webapp.
There, I added a comment that we need <provided> scope to avoid the "multiple locatons" warnings.
https://github.com/eclipse/lyo/pull/500/files#diff-320f130c62507465657c94504081691c25226c6e24b47de7603e32fc6f34b8afR19

Reply:
great, thanks